File: 2022-2023/sourcecode/lab/lab12/ModulesBasicsJava/app/src/main/java/pl/edu/uwr/pum/modulesbasicsjava/computer/Computer.java
package pl.edu.uwr.pum.modulesbasicsjava.computer;

import pl.edu.uwr.pum.modulesbasicsjava.computer.components.CPU;
import pl.edu.uwr.pum.modulesbasicsjava.computer.components.GPU;

public class Computer {
    private GPU gpu;
    private CPU cpu;

    public Computer(GPU gpu, CPU cpu) {
        this.gpu = gpu;
        this.cpu = cpu;
    }

    public String work(){
        return "working";
    }
}

File: 2022-2023/sourcecode/lab/lab12/ModulesBasicsKotlin/app/src/main/java/pl/edu/uwr/pum/modulesbasicskotlin/computer/Computer.kt
package pl.edu.uwr.pum.modulesbasicskotlin.computer

import pl.edu.uwr.pum.modulesbasicskotlin.computer.components.CPU
import pl.edu.uwr.pum.modulesbasicskotlin.computer.components.GPU

class Computer(val gpu: GPU, val cpu: CPU) {
    fun work():String{
        return "working"
    }
}